<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>PdfMerger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../.designer/backup/resources/icon.png</normaloff>
    <normalon>../../../../../../../.designer/resources/icon.png</normalon>../../../../../../../.designer/backup/resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="Line" name="line_11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mergeSelectedButton">
        <property name="text">
         <string>Merge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mergePdfButton">
        <property name="text">
         <string>Merge PDF Items</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mergeCompressedButton">
        <property name="text">
         <string>Merge Compressed Items</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="compressSelectedButton">
        <property name="text">
         <string>Compress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="compressAllButton">
        <property name="text">
         <string>Compress All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeAllButton">
        <property name="text">
         <string>Remove All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addFolderButton">
        <property name="text">
         <string>Add Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addItemButton">
        <property name="text">
         <string>Add Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_12">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>PDF Items</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Compressed PDFS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Merged PDFS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QListWidget" name="pdfListView">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="compressedListView">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="mergedListView">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionDelete_Item"/>
    <addaction name="actionDelete_All"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Merge</string>
    </property>
    <addaction name="actionMerge_2"/>
    <addaction name="actionMerge_All"/>
   </widget>
   <widget class="QMenu" name="menuCompress">
    <property name="title">
     <string>Compress</string>
    </property>
    <addaction name="actionCompress_Selected"/>
    <addaction name="actionCompress_All"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuCompress"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="text">
    <string>Merge</string>
   </property>
  </action>
  <action name="actionMerge_2">
   <property name="text">
    <string>Merge Selected</string>
   </property>
  </action>
  <action name="actionMerge_All">
   <property name="text">
    <string>Merge All</string>
   </property>
  </action>
  <action name="actionCompress_Selected">
   <property name="text">
    <string>Compress Selected</string>
   </property>
  </action>
  <action name="actionCompress_All">
   <property name="text">
    <string>Compress All</string>
   </property>
  </action>
  <action name="actionDelete_Item">
   <property name="text">
    <string>Remove Selected</string>
   </property>
  </action>
  <action name="actionDelete_All">
   <property name="text">
    <string>Remove All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
